en/user_guide/content_type.rst

```diff
@@ -0,0 +1,94 @@
+Content Types
+=============
+
+In Open Orchestra Contents are typed, which means that every content has meaning, that it belongs
+to a family: an "Article" content is different from a "Customer" content. For instance, an article
+requires to contribute a title, a teaser, a picture and a text, while a customer requires instead
+a name, a first name, an email address and a phone number. Open Orchestra offers a RAD approach of
+the Content Types that does not need to write any code for this part: from the description of the
+Content Type to Content editing forms through storage in the database, everything is natively
+supported.
+
+
+Listing
+-------
+
+The list of existing Content Types is available in Administration part, under the Content Type entry.
+This list includes a row per Content Type and indicates its internal id, its name in the language of
+the Back Office, the number of the most advanced version as well as the status of this version.
+
+.. image:: ../../images/content_type_list.png
+
+
+Creation
+--------
+
+To create a new Content Type, from the list of Content Types, click the "Add" button. The Content Type
+creation form is displayed.
+
+.. image:: ../../images/content_type_create_1.png
+
+In the case we want to define the Content Type "Article", we should then contribute the following fields:
+
+* **Content Type id**: that unique identifier allows Open Orchestra to distinguish our "Article" among
+others Content Type. Indicate "article"
+
+* **Name in all the languages**: this is the name of the Content Type, displayed to contributors. It must
+be translated into the different languages ​​of the Back Office. Indicate "Article" for both English and French
+
+* **Status**: determines the status in which to create the Content Type. Choose "draft" for this
```

Comment:
I understand your point of view, but in this tutorial it would be confusing to talk in that way as it refers to nothing concrete. Moreover, it doesn't matter that the status is initial or not, I only need to choose one of the choices.
I'll rewrite this part by describing possible choices in my cases and the choose one.